xbmc/utils/StringUtils.h

```diff
@@ -54,9 +56,26 @@ class StringUtils
   \param ... variable number of value type arguments
   \return Formatted string
   */
-  static std::string Format(PRINTF_FORMAT_STRING const char *fmt, ...) PARAM1_PRINTF_FORMAT;
+  template<typename... Args>
+  static std::string Format(const std::string& fmt, const Args&... args)
```

Comment:
I think the proper way to handle this is with `Args&&... args` and then calling `fmt::format(fmt, std::forward<Args>(args)...);` to just pass along whatever pointer/const/ref/rvalue specifier that was passed in.

example https://github.com/fmtlib/fmt/blob/master/fmt/format.h#L3950